googleapis_auth/lib/auth_io.dart

```diff
@@ -115,13 +115,16 @@ Future<AutoRefreshingAuthClient> clientViaApplicationDefaultCredentials({
 /// {@macro googleapis_auth_close_the_client}
 /// {@macro googleapis_auth_not_close_the_baseClient}
 /// {@macro googleapis_auth_listen_port}
+///
+/// {@macro googleapis_auth_custom_post_auth_page}
```

Comment:
You need to define this macro somewhere!

Reply:
And where do I do that?

Reply:
> You need to define this macro somewhere!

I just never done it before